docs/overview/cmake-can-not.rst

```diff
@@ -36,10 +36,10 @@ way: if you want to do some nasty non-standard thing then probably you should
 stop.  If you think it is something important and useful, then it might be
 quite useful for other :ref:`CMake <CMake>` users too. In this case you need to
 think about implementing new feature **in CMake itself**.  :ref:`CMake <CMake>`
-is open-source project written in C++, I hope this language has enough power
-for you :) Also you can discuss this problem in `CMake mailing-list
+is open-source project written in C++, and additional features are always being added.
+Additionaly,  you can discuss this problem in the `CMake mailing-list
```

Comment:
I think here are too many words with root "add": additional, added, additionaly. It would be nice to reword this part.

Reply:
Updated.